src/crackmes/hackthebox/find-the-easy-pass/fire-up-the-debugger.md

```diff
@@ -0,0 +1,83 @@
+# Fire up the debugger
+
+The next step will be to put a breakpoint at 0x0045412b,
+this address contains probably the right password or our.
```

Comment:
I think you should get all of these addresses/offsets inside single backticks.

Reply:
Sure, I will do.